index.src.html

```diff
@@ -3825,6 +3852,8 @@
         and {{VideoFrame/[[display height]]}}.
     5. Assign `null` to |frame|'s {{VideoFrame/[[duration]]}} and
         {{VideoFrame/[[timestamp]]}}.
+    6. Assign `null` to |frame|'s {{VideoFrame/[[metadata]]}}.
```

Comment:
Probably should be an empty dictionary, since it's being returned directly?

Reply:
I used `null` for that purpose but it is clearer to just set it to a new VideoFrameMetadata.

Reply:
Done